.eslintrc.json

```diff
@@ -31,7 +31,11 @@
         "no-console": ["warn", { "allow": ["error", "deprecate", "warn"] }],
         "eol-last": ["error", "always"],
         "padding-line-between-statements": ["warn", { "blankLine": "always", "prev": "*", "next": "return" }],
-        "space-before-function-paren": ["error", "never"],
+        "space-before-function-paren": ["error", {
+            "anonymous": "never",
+            "async": "always",
```

Comment:
Is this so you don't end up with `async() =>`?

Reply:
Yes. The others are to preserve the existing rules enforcing spacing in parentheses (which we've discussed moving away from, but that _definitely_ is out of scope here 🙂 ).